### DISCUSSION TITLE
What are the various components in the retro display?

### DISCUSSION THREAD

**Discussion by kubark42** [iltis42/XCVario#249] [Q&A]
The manual does not currently define what the various components on the retro display are. Some of the elements can be figured out from Sections 6 and 7, but others are a little unclear.

<img width="662" alt="Screenshot 2023-08-09 at 8 34 07 AM" src="https://github.com/iltis42/XCVario/assets/1118185/f7c0c84e-8001-43d0-a33b-8e9649f41749">

It would be nice to have defined names for all of the various elements. I've started below, could someone help out by giving the rest?

1. OAT
2. Wind direction / Wind speed (units given in top left)
3. Airspeed (TAS/CAS/IAS can be defined in settings)
4. Speed to fly vs thermalling icon (speed to fly is featured)
5. Wireless connection (bluetooth is featured)
6. ???
7. Flaps icon 
8. Flaps matrix
9. Currently-recommended flaps setting
10. Speed to fly chevron.
11. Average vario reading
12. Altimeter setting
13. Battery health
14. Altimeter measurement
15. ???
16. ???
17. McReady value
18. ??? 
19. Instantaneous vario reading
20. ??? (Maybe net climb?)
21. ???
22. Vario base units

With a well-defined name, the manual can consistently refer to the objects, even if the object changes meaning (e.g. netto vs relative mode).

**Comment by pdk42** [comment]
I assume that 15 and 16 are showing the wind direction in relation to the glider.

**Comment by pdk42** [comment]
And according to section 6.2 of the manual, 20 is indeed the mean climb rate which is a smoothed value of climbs over the last 45 mins.  Section 10.7.10 explains how the mean climb is calculated.

Text below:

"The mean climb is represented as a colored symbol in the form of a small route within
the variometer scale and moves up accordingly. Climb values that are too low and that
you do not want to include in the core climb can be hidden in the Vario dialog. The
medium rise serves to provide the best possible suggestion for the MacCready value.

The icon changes color to green if the average climb has improved by more than 0.2
m/s since the last measurement and to red if the climb has decreased by 0.2 m/s. If 
this improves significantly, more precisely by more than 20%, the upper part of the route extends into an arrow as
shown on the right. In the opposite case, with a clear deterioration lengthens the lower part of the then red symbol
shown."
